packages/react/src/components/Tree/TreeItem.tsx

```diff
@@ -52,21 +63,22 @@ export interface TreeItemState {
   open?: boolean
 }
 
-class TreeItem extends AutoControlledComponent<ReactProps<TreeItemProps>, TreeItemState> {
+class TreeItem extends UIComponent<ReactProps<TreeItemProps>, TreeItemState> {
```

Comment:
Why this change is necessary? This means that controlled mode will not work **(!)**, see example:
https://codesandbox.io/s/845j354jx2?module=/example.js

Please revert it.